src/com/company/Main.java

```diff
@@ -2,7 +2,22 @@
 
 public class Main {
 
-    public static void main(String[] args) {
+    public long faktorial (int n){
```

Comment:
please add comment to each method that you use!

Reply:
OK